src/Microsoft.VisualBasic.Core/tests/InteractionTests.cs

```diff
@@ -98,5 +114,44 @@ private static IEnumerable<object[]> IIf_TestData()
             yield return new object[] { false, 3, "str", "str" };
             yield return new object[] { true, 3, "str", 3 };
         }
+
+        [Theory]
+        [InlineData(0, 1, 100, 10, "   :  0")]
+        [InlineData(1, 1, 100, 10, "  1: 10")]
+        [InlineData(15, 1, 100, 10, " 11: 20")]
+        [InlineData(25, 1, 100, 10, " 21: 30")]
+        [InlineData(35, 1, 100, 10, " 31: 40")]
+        [InlineData(45, 1, 100, 10, " 41: 50")]
+        [InlineData(5001, 1, 10000, 100, " 5001: 5100")]
+        public void Partition(long Number, long Start, long Stop, long Interval, string expected)
```

Comment:
>public [](start = 8, length = 6)

Please include some tests where the upper part is empty.

Reply:
Please test cases where the resulting lower and upper parts are not padded with spaces.

---
In reply to: [304679914](https://github.com/dotnet/corefx/pull/39574#discussion_r304679914) [](ancestors = 304679914)

Reply:
Tests added as suggested.

Reply:
Thanks. 

Consider adding some tests with `long.MaxValue` such as the following. (Sorry for not suggesting these earlier.)

Valid:
```
[InlineData(1, 0, 1, long.MaxValue, "0:1")]
[InlineData(1, 0, long.MaxValue / 2, long.MaxValue / 2, "   0:...")]
```

OverflowException:
```
[InlineData(1, 0, long.MaxValue, 1)]
[InlineData(1, 0, long.MaxValue, long.MaxValue)]
[InlineData(long.MaxValue - 1, long.MaxValue - 1, long.MaxValue, 1)]
```

---
In reply to: [304726847](https://github.com/dotnet/corefx/pull/39574#discussion_r304726847) [](ancestors = 304726847)

Reply:
Added those tests and few more